packages/core/src/theme/__snapshots__/theme.spec.ts.snap

```diff
@@ -10,11 +10,11 @@ Object {
     "0 0 2px 0 rgba(0,0,0,.08),0 8px 32px 0 rgba(0,0,0,.16)",
   ],
   "breakpoints": Array [
-    "32em",
-    "40em",
-    "48em",
-    "64em",
-    "80em",
+    "32",
```

Comment:
same here

Reply:
I totally missed this, I'll have to look into what's causing it